ci-operator/config/netobserv/network-observability-cli/netobserv-network-observability-cli-main.yaml

```diff
@@ -0,0 +1,44 @@
+base_images:
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "9"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  to: network-observability-cli
+promotion:
+  to:
+  - namespace: netobserv
+    tag: ci
+releases:
+  initial:
+    integration:
+      name: "4.16"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.16"
+      namespace: ocp
```

Comment:
I figured we don't need multiple releases here, just one single nightly release should be sufficient, see my PR: https://github.com/openshift/release/pull/50006/files

Reply:
Thanks for the input, I'll update this in my PR too :+1:

Reply:
https://github.com/openshift/release/pull/50198/commits/f0c22b076d70abf8378bc079896950bc75313b08